Unmasked/Unmasked View/View Controllers/UnmaskedViewController.swift

```diff
@@ -0,0 +1,259 @@
+    @IBOutlet private weak var powerBorder: UIImageView!
+    
+    public var hero: SuperheroResponseModel?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        initialSetup()
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "unmaskedToCollection" {
+            super.prepare(for: segue, sender: sender)
+            let destination = segue.destination as! HeroCollectionViewController
+            destination.heroGroup = heroGroup
+        }
+    }
+    
+    private func initialSetup() {
+        guard let chosenHero = self.hero else {
+            return
+        }
+        unmaskedViewModel = UnmaskedViewModel(hero: chosenHero,
+                                              heroInfoGroup: .powerstats,
+                                              delegate: self)
+        heroImageView.loadImage(with: chosenHero.image.url)
+        heroNameLabel.text = chosenHero.name
+        infoScreenTitleLable.text = unmaskedViewModel?.heroInfoGroup.rawValue
+        setupInfoView()
+        menuSetup()
+    }
+    
+    private func menuSetup() {
+        rotateMenuButtons()
+        rotateMenuBorders()
+        autoShrinkAllButtonText()
+    }
+    
+    private func rotateMenuButtons() {
+        appearanceButton.transform = CGAffineTransform(rotationAngle: CGFloat.pi/6)
+        biographyButton.transform = CGAffineTransform(rotationAngle: -(CGFloat.pi/6))
+        workButton.transform = CGAffineTransform(rotationAngle: CGFloat.pi/6)
+        connectionButton.transform = CGAffineTransform(rotationAngle: -(CGFloat.pi/6))
+    }
+    
+    private func rotateMenuBorders() {
+        appearanceBorder.transform = CGAffineTransform(rotationAngle: CGFloat.pi/6)
+        biographyBorder.transform = CGAffineTransform(rotationAngle: -(CGFloat.pi/6))
+        workBorder.transform = CGAffineTransform(rotationAngle: CGFloat.pi/6)
+        connectionBorder.transform = CGAffineTransform(rotationAngle: -(CGFloat.pi/6))
+    }
+    
+    private func autoShrinkAllButtonText() {
+        autoShrinkButtonText(btn: powerButton)
+        autoShrinkButtonText(btn: biographyButton)
+        autoShrinkButtonText(btn: appearanceButton)
+        autoShrinkButtonText(btn: workButton)
+        autoShrinkButtonText(btn: connectionButton)
+    }
+    
+    private func autoShrinkButtonText(btn: UIButton) {
+        btn.titleLabel?.adjustsFontSizeToFitWidth = true
+        btn.titleLabel?.minimumScaleFactor = CGFloat(0.3)
+        btn.titleLabel?.lineBreakMode = NSLineBreakMode.byClipping
+    }
+    
+    private func setupInfoView() {
+        if heroInfoView.subviews.count != 0 {
+            heroInfoView.subviews[0].removeFromSuperview()
+        }
+        
+        switch unmaskedViewModel?.currentInfoGroup {
+        case .powerstats:
+            let view = PowerstatsView.instanceFromNib() as! PowerstatsView
+            view.setup(pStat: unmaskedViewModel!.chosenHero!.powerstats)
```

Comment:
Force unwrapping also you can break this down more in the `ViewModel` this could just be `unmaskedViewModel.powerStats`